test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -30,11 +30,19 @@
 
 apply plugin: 'opensearch.java'
 
+versions << [
+  'jetty': '9.4.51.v20230217'
```

Comment:
Please add it to `version.properties` so we have a central place to manage those